### DISCUSSION TITLE
Hidden text behind elements

### DISCUSSION THREAD

**Discussion by klimach** [UglyToad/PdfPig#734] [Q&A]
Is it possible to identify the text behind the elements? I collect information from pdf files, identifying only the text that lies on the drawn shapes, but there are cases where the text is behind these shapes, and I would like to ignore it. Any ideas would be appreciated
![image](https://github.com/UglyToad/PdfPig/assets/41283428/ef4538bb-7e67-411c-9b35-7f474f4f9030)
![image](https://github.com/UglyToad/PdfPig/assets/41283428/72372ea3-f30a-4b43-9d72-a3d04864f67f)

**Comment by topcat30** [comment]
Im also wanting a way to identify background text. in my case in want to ignore it

**Reply by topcat30** [reply]
i got around my issue by ignoring text that is not rotation 0
